conf/modules.config

```
@@ -543,7 +543,12 @@ process {
     }
 
     withName: TAXPASTA_MERGE {
-        ext.args =  { "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}" }
+        ext.args =  {
+            [
+                "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}",
+                params.taxpasta_add_taxonomy ? "--taxonomy" : "",
```

```suggestion
                params.taxpasta_add_taxonomy ? "--taxonomy ${params.taxpasta_add_taxonomy}" : "",
```

I do wonder if the flag is maybe misleading to it's purpose, add_taxonomy sounds like a command to _do_ something, but rather the flag is rather just asking for a path... what about `--taxapasta_taxonomyfiles` or something?

---

I find files a bit misleading since it should be a directory. So maybe either `--taxpasta_taxonomy` or `--taxpasta_taxonomy_dir`?

---

I would prefer `--taxpasta_taxonomy_dir` then.